packages/core/data-transfer/types/providers.d.ts

```diff
@@ -34,5 +34,6 @@ export interface IDestinationProvider extends IProvider {
   getLinksStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
   getMediaStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
   getConfigurationStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
+  getSchemas?(): any;
   getSchemasStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
```

Comment:
```suggestion
  getLinksStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
  getMediaStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
  getConfigurationStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
  getSchemasStream?(): NodeJS.WritableStream | Promise<NodeJS.WritableStream>;
  
  // Provide the schema to use for schemaMatching validation
  getSchemas?(): any;
```

Also, can we use an already existing Strapi typing to put for these to make sure it only has the expected keys? Not super important, but it could be nice to have something.